pkg/devfile/adapters/docker/component/utils.go

```diff
@@ -10,9 +14,12 @@ import (
 	adaptersCommon "github.com/openshift/odo/pkg/devfile/adapters/common"
 	"github.com/openshift/odo/pkg/devfile/adapters/docker/utils"
 	versionsCommon "github.com/openshift/odo/pkg/devfile/parser/data/common"
+	"github.com/openshift/odo/pkg/envinfo"
 	"github.com/openshift/odo/pkg/log"
 )
 
+var LocalhostIP = "127.0.0.1"
```

Comment:
Have you tested this on both Windows, macOS and Linux platforms? I have a feeling that there may be an issue with one of the OS'

Reply:
Tested on Windows, macoS and Linux. All worked as expected.